### ISSUE TITLE
Pixelated bars since 3.x

### ISSUE THREAD

**Issue by shllg** [fgnass/spin.js#355]
I updated spin.js lately from 2.3.2 to 3.1.0

Before, the spinner had clean rectangels as bars. Now with the new version they're looking pixelated/without smooth edges.

![bildschirmfoto 2018-03-15 um 12 07 36](https://user-images.githubusercontent.com/5073881/37459952-02270bec-284a-11e8-9e24-b72f0605b16d.png)

I tried different options to get rid of those edges - without success.

**Comment by shllg** [comment]
Before it looked like that:

![bildschirmfoto 2018-03-15 um 12 14 08](https://user-images.githubusercontent.com/5073881/37460092-743b2b0a-284a-11e8-924b-458f69a4c4fb.png)

**Comment by theodorejb** [comment]
Thanks for the report. It looks like the two screenshots are using different settings (number of lines, length, radius, etc.), which makes it harder to compare them. Can you share the settings used for both screenshots? Also, what browser is this from? Do you notice pixelation in other browsers too?

**Comment by shllg** [comment]
Yes the settings are slightly different, but changing them doesn't fix the problem.

System: MacOS - Chrome / Firefox / Safari all with the same issue

![bildschirmfoto 2018-03-18 um 20 28 36](https://user-images.githubusercontent.com/5073881/37570101-947d1196-2aeb-11e8-8fa7-1d4bad8b3810.png)

Settings:

```coffeescript
lines: 11             # The number of lines to draw
length: 12            # The length of each line
width: 2              # The line thickness
radius: 8             # The radius of the inner circle
scale: 1              # Scales overall size of the spinner
corners: 1            # Corner roundness (0..1)
color: '#000'         # #rgb or #rrggbb or array of colors
opacity: 0.25         # Opacity of the lines
rotate: 0             # The rotation offset
direction: 1          # 1: clockwise, -1: counterclockwise
speed: 1              # Rounds per second
trail: 60             # Afterglow percentage
fps: 20               # Frames per second when using setTimeout() as a fallback for CSS
zIndex: 2e9           # The z-index (defaults to 2000000000)
className: 'spinner'  # The CSS class to assign to the spinner
top: '50%'            # Top position relative to parent
left: '50%'           # Left position relative to parent
shadow: false         # Whether to render a shadow
hwaccel: false        # Whether to use hardware acceleration
position: 'absolute'  # Element positioning
```

**Comment by theodorejb** [comment]
I don't have a Mac to test with, but I noticed that rendering the lines with a transparent shadow seems to help in Chrome on Windows. Try setting `shadow: '0 0 1px transparent'` and see if it makes a difference.

**Comment by shllg** [comment]
Not i just blurs the edges a little bit. But the problem itself is still there. 

![bildschirmfoto 2018-03-22 um 11 54 11](https://user-images.githubusercontent.com/5073881/37766657-e22a9312-2dc7-11e8-8f23-2cb22c17f0fa.png)

(Again different settings but i think you can still see the issue)

**Comment by shllg** [comment]
Okay, i'm sorry - seems like i were wrong. I ran some comparison with the versions:

![bildschirmfoto 2018-03-22 um 13 29 44](https://user-images.githubusercontent.com/5073881/37770944-582ee8b6-2dd6-11e8-9a91-6adef3b1671f.png)

Same settings except of the last one with you recommended shadow setting.
